### ISSUE TITLE
Nav.astro redudant to Header.astro?

### ISSUE THREAD

**Issue by pizzlemank** [treefarmstudio/odyssey-theme#43]
As titled. `Header.astro` includes the slot NAV and the styling already. Or did I miss something.
Thanks.

**Comment by AdamGEmerson** [comment]
Yes, I also found this to be true.  The `Nav.astro` component doesn't appear to be used in the template.
